lib/livebook/notebook_manager.ex

```diff
@@ -243,8 +243,11 @@ defmodule Livebook.NotebookManager do
         _ -> %{}
       end
 
+    # TODO: Load file system from all hubs
+    personal = Livebook.Hubs.fetch_hub!(Livebook.Hubs.Personal.id())
```

Comment:
The issue is that we insert the personal hub after `NotebookManager` starts, we are won't have this issue, so let's just do this:

```suggestion
    file_systems =
      Livebook.Storage.all(:file_systems)
      |> Enum.sort_by(& &1.bucket_url)
      |> Enum.map(fn fields -> FileSystems.load(fields.type, fields) end)
```